packages/graphql-transformer-common/src/definition.ts

```diff
@@ -112,6 +113,13 @@ export function isScalarOrEnum(type: TypeNode, enums: EnumTypeDefinitionNode[])
   }
 }
 
+export function isEnum(type: TypeNode, document: DocumentNode) {
```

Comment:
It looks like the `graphql` module has a `isEnumType()` function. Would that work for your purposes?

Reply:
It doesn't seem to be working for FieldDefinition types. I have tried different options before creating a new function.